### DISCUSSION TITLE
Simulation parameter in channel flow

### DISCUSSION THREAD

**Discussion by jushi-123** [xcompact3d/Incompact3d#310] [Q&A] [answered]
Hello Dr. Laizet and the Incompact3d Development Team,
I have some doubt in setting the parameter of turbulent channel flow,especially the steps of rotation and the rotation speed. If the Re number is fixed and the beta is changed to refine the near wall grid. Is it necessary to change this two parameters?Is there any tips in setting these two parameters?
I would be grateful for any suggestions from the Incompact3d Development Team that could assist me in this regard.

**Comment by slaizet** [comment, marked as answer]
Hi,
The rotation is just here at the start of the simulation to accelerate the transition from a laminar channel to a turbulent channel flow. It only needs to be active for few thousand time steps (at the start of the simulation). There is no need to change the other parameters.
Best,
Sylvain

**Reply by slaizet** [reply]
Hi, there should be no dependence of dt for the final statistical steady state (and you need to keep beta constant for a given simulation).
Best,
Sylvain
